### PR TITLE
Added a link to OTR messaging in the drop down menu using modals, which is where OTR messaging would be linked after it is implemented properly

### PR DESCRIPTION
**Pull request by fallacy321**

### CODE REVIEW

File: defaults.yml [hunk 1]
```diff
@@ -23,9 +23,10 @@ xmpp:
     handled: false
     unhandled: false
   port: 5222
-  host: localhost
+  domain: jappix.com
```
Comment (tonghuix) [inline defaults.yml]: it is no needs to change it.

File: defaults.yml [hunk 2]
```diff
@@ -23,9 +23,10 @@ xmpp:
...
+
   tls:
-    enable: false
+    enable: true
```
Comment (tonghuix) [inline defaults.yml]: at this point, I suppose tls is not a necessary option, but it is ok. We should regenrate the website key for tls

File: templates/chat.html [hunk 3]
```diff
@@ -113,5 +117,6 @@ <h2 class="lcb-rooms-browser-heading">
         <% include 'includes/modals/xmpp.html' %>
         <% include 'includes/modals/tokens.html' %>
         <% include 'includes/modals/upload.html' %>
+        <% include 'includes/modals/a.html' %>
```
Comment (tonghuix) [inline templates/chat.html]: a.html, it is not a good name. please change it.

File: templates/includes/js/message.html [hunk 4]
```diff
@@ -11,7 +11,8 @@
                             {{owner.displayName}}
                         </span>
                         <span class="lcb-message-username">
-                            @{{owner.username}}
+                            @{{owner.username}} 
+                            {{#if mentioned}}<font color="grey"><strong>ACTIVITY ! ! </strong></font> {{/if}}
```
Comment (tonghuix) [inline templates/includes/js/message.html]: I suggest divided this pr to two, one is mentioned notice another is OTR menu.

File: defaults.yml (hunk 1 above)
Comment (fallacy321) [inline defaults.yml]: done

File: defaults.yml (hunk 2 above)
Comment (fallacy321) [inline defaults.yml]: done

File: templates/includes/js/message.html (hunk 4 above)
Comment (fallacy321) [inline templates/includes/js/message.html]: I tried maintaining a separate commit for this, if this is what it meant. Please let me know, if I am wrong :)

File: templates/chat.html (hunk 3 above)
Comment (fallacy321) [inline templates/chat.html]: Updated

**Pull request merged**